pkg/util/openshift.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
+	"os"
+	"strings"
+)
+
+var (
+	openshift *bool
+)
+
+func IsOpenshift() bool {
+	if openshift == nil {
+		b := detectOpenshift()
+		openshift = &b
+	}
+	return *openshift
+}
+
+func detectOpenshift() bool {
+
+	value, ok := os.LookupEnv("ENMASSE_OPENSHIFT")
```

Comment:
We should detect it by querying the API, like in https://github.com/EnMasseProject/enmasse/blob/master/keycloak-controller/src/main/java/io/enmasse/keycloak/controller/KeycloakController.java#L170 .

Reply:
I wanted to ask you about, sounds like a good idea.

Reply:
Done.